src/SourceGenerators/Uno.UI.SourceGenerators/BindableTypeProviders/BindableTypeProvidersGenerationTask.cs

```diff
@@ -67,56 +67,48 @@ internal void Generate(GeneratorExecutionContext context)
+						);
+						_assemblyName = context.GetMSBuildPropertyValue("AssemblyName");
 
-							_defaultNamespace = context.GetMSBuildPropertyValue("RootNamespace");
-							_namedSymbolsLookup = context.Compilation.GetSymbolNameLookup();
+						_defaultNamespace = context.GetMSBuildPropertyValue("RootNamespace");
+						_namedSymbolsLookup = context.Compilation.GetSymbolNameLookup();
 
-							_bindableAttributeSymbol = FindBindableAttributes();
-							_dependencyPropertySymbol = context.Compilation.GetTypeByMetadataName(XamlConstants.Types.DependencyProperty);
-							_dependencyObjectSymbol = context.Compilation.GetTypeByMetadataName(XamlConstants.Types.DependencyObject);
+						_bindableAttributeSymbol = FindBindableAttributes();
+						_dependencyPropertySymbol = context.Compilation.GetTypeByMetadataName(XamlConstants.Types.DependencyProperty);
+						_dependencyObjectSymbol = context.Compilation.GetTypeByMetadataName(XamlConstants.Types.DependencyObject);
 
-							_javaObjectSymbol = context.Compilation.GetTypeByMetadataName("Java.Lang.Object");
-							_nsObjectSymbol = context.Compilation.GetTypeByMetadataName("Foundation.NSObject");
-							_nonBindableSymbol = context.Compilation.GetTypeByMetadataName("Windows.UI.Xaml.Data.NonBindableAttribute");
-							_resourceDictionarySymbol = context.Compilation.GetTypeByMetadataName("Windows.UI.Xaml.ResourceDictionary");
-							_currentModule = context.Compilation.SourceModule;
+						_javaObjectSymbol = context.Compilation.GetTypeByMetadataName("Java.Lang.Object");
+						_nsObjectSymbol = context.Compilation.GetTypeByMetadataName("Foundation.NSObject");
+						_nonBindableSymbol = context.Compilation.GetTypeByMetadataName("Windows.UI.Xaml.Data.NonBindableAttribute");
+						_resourceDictionarySymbol = context.Compilation.GetTypeByMetadataName("Windows.UI.Xaml.ResourceDictionary");
+						_currentModule = context.Compilation.SourceModule;
 
-							var modules = from ext in context.Compilation.ExternalReferences
-										  let sym = context.Compilation.GetAssemblyOrModuleSymbol(ext) as IAssemblySymbol
-										  where sym != null
-										  from module in sym.Modules
-										  select module;
+						var modules = from ext in context.Compilation.ExternalReferences
+									  let sym = context.Compilation.GetAssemblyOrModuleSymbol(ext) as IAssemblySymbol
+									  where sym != null
+									  from module in sym.Modules
+									  select module;
 
-							modules = modules.Concat(context.Compilation.SourceModule);
+						modules = modules.Concat(context.Compilation.SourceModule);
 
-							context.AddSource("BindableMetadata", GenerateTypeProviders(modules));
+						context.AddSource("BindableMetadata.g.cs", GenerateTypeProviders(modules));
 
-							GenerateLinkerSubstitutionDefinition();
-						}
-						else
-						{
-							context.AddSource("BindableMetadata", $"// validPlatform: {validPlatform} designTime:{isDesignTime} isApplication:{isApplication}");
```

Comment:
This is not necessary. In this code path, we already know that `validPlatform` is true, and `isDesignTime` is false, and `isApplication` is false. So this sort of logging doesn't add any useful information.

I guess this generated source was the problematic one since the source added in the other code path is already marked as auto-generated.